src/chart/pie/pieLayout.ts

```diff
@@ -30,9 +30,9 @@ const RADIAN = Math.PI / 180;
 function getViewRect(seriesModel: PieSeriesModel, api: ExtensionAPI) {
     return layout.getLayoutRect(
         seriesModel.getBoxLayoutParams(), {
-            width: api.getWidth(),
-            height: api.getHeight()
-        }
+        width: api.getWidth(),
+        height: api.getHeight()
+    }
```

Comment:
I think these three lines (or perhaps this file?) should not be changed.

Reply:
fixed